pkg/controller/kieapp/defaults/defaults.go

```diff
@@ -1073,6 +1077,19 @@ func getKafkaConfig(kafka *api.KafkaExtObject) {
 	}
 }
 
+func getKafkaJBPMConfig(kafkaExt *api.KafkaExtObject, kafka *api.KafkaJBPMObject) {
+	// The emitter is enabled only if the kafka extension is enabled
+	if kafkaExt != nil && kafka != nil {
```

Comment:
please check this with guys from jbpm, the configuration through system properties looks like that it may be possible to use only kafka jbpm event emitter

Reply:
Asked to Francisco, waiting a reply

Reply:
Asked to Francisco, are separate things